esphome/components/kalman_combinator/kalman_combinator.cpp

```diff
@@ -0,0 +1,104 @@
+#include "kalman_combinator.h"
+#include "esphome/core/hal.h"
+#include <cmath>
+#include <functional>
+
+namespace esphome {
+namespace kalman_combinator {
+
+void KalmanCombinatorComponent::dump_config() {
+  ESP_LOGCONFIG("kalman_combinator", "Kalman Combinator:");
+  ESP_LOGCONFIG("kalman_combinator", "  Update variance per ms: %f", this->update_variance_);
```

Comment:
Reads better like this, imo:
```suggestion
  ESP_LOGCONFIG("kalman_combinator", "  Update variance: %f per ms", this->update_variance_);
```

Reply:
bf62d38